src/mesh/NodeDB.cpp

```diff
@@ -648,6 +648,26 @@ void NodeDB::loadFromDisk()
     if (state == LoadFileResult::SUCCESS) {
         LOG_INFO("Loaded OEMStore\n");
     }
+
+    // 2.4.X - configuration migration to update new default intervals
+    if (moduleConfig.version < DEVICESTATE_CUR_VER) {
```

Comment:
I wonder if it might be good to hard-code this as `if(moduleConfig.version < 23)`, to prevent it running again in the future, when `DEVICESTATE_CUR_VER` increments again? Optional though!

Reply:
Agree. Real numbers are the way to go.

Reply:
The one time mama said not to use a constant! Good call, guys. 😆